spec/controllers/result_comments_controller_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ResultCommentsController, type: :controller do
+  login_user
+
+  let(:user) { User.first }
+  let!(:team) { create :team, :with_members }
```

Comment:
you are creating team here but `user` is not a member of it  I think. Just FYI

Reply:
:+1: